products/compute/terraform.yaml

```diff
@@ -348,6 +348,10 @@ overrides: !ruby/object:Provider::ResourceOverrides
         exclude: false
       region: !ruby/object:Provider::Terraform::PropertyOverride
         exclude: true
+      addressType: !ruby/object:Provider::Terraform::PropertyOverride
+        diff_suppress_func: 'emptyOrDefaultStringSuppress("EXTERNAL")'
```

Comment:
since this has a default value, does it actually need this diffsuppress?

Reply:
Mentioned IRL: passing in External or "" to the api will return with an empty string no matter what. So looks like it's still needed.